aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteModelFieldTypeConverter.java

```diff
@@ -98,8 +98,15 @@ public static Object convertRawValueToTarget(
                 boolean booleanValue = (boolean) value;
                 return booleanValue ? 1L : 0L;
             case MODEL:
+                if (value instanceof Map) {
+                    Map<?, ?> map = (Map<?, ?>) value;
```

Comment:
(@richardmcclellan The previous build failure was that I didn't have a space after `,` in the `<?,?>`.)